docs/specs/forecast-data.md

```diff
@@ -28,8 +24,10 @@ Additional attributes, in addition to the required and suggested ones above, may
 - `longitude` (`float32`): `np.arange(-180, 180, 0.25)`, EPSG:4326
 - `latitude` (`float32`): `np.arange(-90, 90, 0.25)`, EPSG:4326
 - `flight_level` (`int16` or `int32`): `[270, 280, 290, 300, 310, 320, 330, 340, 350, 360, 370, 380, 390, 400, 410, 420, 430, 440]`, hectofeet [^flightlevels]
-- `time` (`int32` or `int64`): [CF compatible time coordinates](https://cfconventions.org/cf-conventions/cf-conventions#time-coordinate). Must have `units` and `calendar` variable attributes. e.g.
+- `time` (`int32` or `int64`): [CF compatible time coordinates](https://cfconventions.org/cf-conventions/cf-conventions#time-coordinate).
+- `forecast_reference_time` (`int32` or `int64`): The forecast reference time is the "data time", i.e. the time at which the meteorological model was executed for a given set of forecast times.`
 
+💡 Both `time` and `forecast_reference_time` must have `units` and `calendar` variable attributes. e.g.
```

Comment:
@mlshapiro & @zebengberg, I think this section of the data contract could use more elaboration. Up to you to decide how best to phrase...

My concern is that this requirement, as phrased is a bit confusing.
xarray, when reading in a netCDF, will _not_ show a `units` or `calendar` attr on `time`.  

In inspecting an existing grid netcdf from our v0 API, I see that these attrs are set when inspecting the file with `ncdump`.

When loaded with xarray, however, the attrs are scrubbed from `time`. I see it is automagically set when serializing to netCDF, and deserializing from netCDF.

Furthermore, the `units` reference value is not static; it is dynamically set, which is confusing for folks who are used to e.g. epoch time with a static reference of 1970-01-01...

Reply:
It is written confusingly, but this is how its written in the [CF Convention](https://cfconventions.org/cf-conventions/cf-conventions#time-coordinate)

I will re-arrange this section to hopefully be a little more clear.

While we use xarray for everything, many others won't, so it should be true to the CF convention. Meterologists will be very comfortable with this format, but its true that developers will not be as versed in CF vagaries..

Reply:
For future reference, the CF convention could support unix time if desired. In the `xarray` paradigm, you'd convert the time coordinate to an `int64` representing seconds since 1970-01-01 00:00:00 and then write a `units` attr that days `"seconds since 1970-01-01 00:00:00"`, e.g.

```
ds["time"] = ((ds["time"] - np.datetime64('1970-01-01T00:00:00')) / np.timedelta64(1, 's')).astype(np.int64)
ds["time"].attrs["units"] = "seconds since 1970-01-01 00:00:00"
```

Then write that out. Calendar is still relevant, so its worth saying its required.

Reply:
Updated the [Dimensions](https://github.com/contrailcirrus/contrail-forecast/blob/refactor/v1-signature/docs/specs/forecast-data.md#dimensions) section and added a little [time dimensions](https://github.com/contrailcirrus/contrail-forecast/blob/refactor/v1-signature/docs/specs/forecast-data.md#time-dimensions) explainer.

Reply:
clear ty!

could maybe use a tldr one-liner at the head of our diff along the lines of:
```
Xarray will automatically include a `units` and `calendar` attribute on all time-like variables when writing to a netCDF file.
{details here}
```